### PR TITLE
CORE-5088: Add Avro schema for consensual ledger persistence requests

### PR DESCRIPTION
**Pull request by omfR3**
Add Avro schema (PersistTransaction.avsc) for sending requests to the entity message processor in the DB worker. Currently the exact same as `PersistRequest`, but we will be adding more stuff later and it needs to be handled with custom code on the receiving side.

* Related: [ledger persistence PR](https://github.com/corda/corda-runtime-os/pull/1883) in corda-runtime-os

### CODE REVIEW

File: data/avro-schema/src/main/resources/avro/net/corda/data/ledger/consensual/FindTransaction.avsc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "type": "record",
+  "name": "FindTransaction",
+  "doc": "Retrieve the specified consensual ledger transaction, specified by id. One of several types of ledger persistence request {@link ConsensualLedgerRequest}",
```
Comment (vlajos) [inline data/avro-schema/src/main/resources/avro/net/corda/data/ledger/consensual/FindTransaction.avsc]:
```suggestion
  "doc": "Retrieve the specified consensual signed transaction, specified by id. One of several types of ledger persistence request {@link ConsensualLedgerRequest}",
```

File: data/avro-schema/src/main/resources/avro/net/corda/data/ledger/consensual/PersistTransaction.avsc [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "type": "record",
+  "name": "PersistTransaction",
+  "doc": "Persist the specified consensual ledger transaction. One of several types of ledger persistence request {@link ConsensualLedgerRequest}",
```
Comment (vlajos) [inline data/avro-schema/src/main/resources/avro/net/corda/data/ledger/consensual/PersistTransaction.avsc]:
```suggestion
  "doc": "Persist the specified consensual signed transaction. One of several types of ledger persistence request {@link ConsensualLedgerRequest}",
```

**Review by blsemo — COMMENTED** [review]
Agree with Lajos's comments.

File: data/avro-schema/src/main/resources/avro/net/corda/data/ledger/consensual/FindTransaction.avsc (hunk 1 above)
Comment (mbrkic-r3) [reply data/avro-schema/src/main/resources/avro/net/corda/data/ledger/consensual/FindTransaction.avsc]: replaced

File: data/avro-schema/src/main/resources/avro/net/corda/data/ledger/consensual/PersistTransaction.avsc (hunk 2 above)
Comment (mbrkic-r3) [reply data/avro-schema/src/main/resources/avro/net/corda/data/ledger/consensual/PersistTransaction.avsc]: replaced

**Review by blsemo — APPROVED** [review]
LGTM

**Pull request merged**